bc-backend/src/main/java/org/jboss/da/bc/impl/ProjectHiearchyCreator.java

```diff
@@ -154,10 +156,24 @@ private ProjectHiearchy toProjectHiearchy(GAV gav) {
         setSCMInfo(project, pomInfo); // scmUrl, useExistingBc
         findExistingBuildConfiguration(project); // bcExists, useExistingBc
         checkInternallyBuilt(project); // internallyBuilt
+        getBuiltVersions(project);
 
         return new ProjectHiearchy(project, false);
     }
 
+    private void getBuiltVersions(ProjectDetail project) {
```

Comment:
I think you can merge this method with ``checkInternallyBuilt`` method (it can be named ``getBuiltVersions``) and use the ``lookupResult`` both for ``availableVersions`` and ``internallyBuilt``.